src/components/Notification/Notfications.jsx

```diff
@@ -8,12 +8,13 @@ export default function Notifications() {
     const dispatch = useDispatch();
 
     useEffect(() => {
+        const timer = (notifications.length != 0 && notifications[0].timeout) ? notifications[0].timeout : 3000

```

Comment:
Store default timer in redux.
Create a selector to get the notification.
Think how we can remove the the need of useEffect (Hint: redux triggers rerender on state change)